libcxx/include/__charconv/traits.h

```diff
@@ -45,12 +45,12 @@ struct _LIBCPP_HIDDEN __traits_base<_Tp, __enable_if_t<sizeof(_Tp) <= sizeof(uin
   ///
   /// The algorithm is based on
   /// http://graphics.stanford.edu/~seander/bithacks.html#IntegerLog10
-  /// Instead of using IntegerLogBase2 it uses __libcpp_clz. Since that
-  /// function requires its input to have at least one bit set the value of
-  /// zero is set to one. This means the first element of the lookup table is
-  /// zero.
+  /// Instead of using IntegerLogBase2 it uses __countl_zero. Previously, it
+  /// used __libcpp_clz. Since that function requires its input to have at
```

Comment:
```suggestion
  /// Instead of using IntegerLogBase2 it uses __countl_zero. Since that function requires its input to have at
```
That's really not interesting IMO. Same below.

Reply:
The old comment "Since that function requires its input to have at least one bit set" only applies to `__libcpp_c{l,t}z`, not to `__count{l,r}_zero`, because `__count{l,r}_zero` already handles all-zero inputs. In this case, since we are not using `__libcpp_c{l,t}z` any more, should I remove this comment completely?